From 751be804e891aec5701a059144e2f5cbfc981b36 Mon Sep 17 00:00:00 2001
From: Andreas Henriksson <andreas@fatal.se>
Date: Thu, 24 Aug 2017 17:52:19 +0200
Subject: [PATCH 08/11] Ship avahi-discover(1), bssh(1) and bvnc(1) also for
 GTK3

These manpages went missing when you disabled gtk2 builds....
---
 man/Makefile.am | 9 +++++----
 1 file changed, 5 insertions(+), 4 deletions(-)

diff --git a/man/Makefile.am b/man/Makefile.am
index 77a27bd..289b942 100644
--- a/man/Makefile.am
+++ b/man/Makefile.am
@@ -56,7 +56,7 @@ man_MANS += \
 	avahi-publish.1 \
 	avahi-set-host-name.1
 
-if HAVE_GTK
+if HAVE_GTK2OR3
 man_MANS += \
 	bssh.1
 endif
@@ -64,12 +64,13 @@ endif
 if HAVE_PYTHON
 man_MANS += \
 	avahi-bookmarks.1
-if HAVE_GTK
+endif
+
+if HAVE_PYGOBJECT
 man_MANS += \
 	avahi-discover.1
 endif
 endif
-endif
 
 if ENABLE_AUTOIPD
 if HAVE_LIBDAEMON
@@ -134,7 +135,7 @@ EXTRA_DIST = \
 if HAVE_DBUS
 
 BSSH_LN =
-if HAVE_GTK
+if HAVE_GTK2OR3
 if HAVE_GLIB
 BSSH_LN += $(LN_S) bssh.1 bvnc.1 &&
 BSSH_LN += $(LN_S) bssh.1 bshell.1 &&
-- 
2.25.2

